### DISCUSSION TITLE
Is inspircd able be linked with charybdis servers?

### DISCUSSION THREAD

**Discussion by walkero-gr** [inspircd/inspircd#1997] [Support]
This is more of a question. Is inspircd able to be linked with charybdis servers? I am trying to do so and I am getting connection timeouts.

I am using spanningtree module to do so, but I am not sure if I am going to the wrong direction.

I am sorry if this was mentioned somewhere and I failed to find it.

**Comment by sadiepowell** [comment]
No. InspIRCd uses a different server protocol to Charybdis (which it should be noted is no longer maintained) so thye can not be linked together. Most IRC servers can only link to other instances of the same software and services with explicit support for them (e.g. Anope, Atheme).

**Comment by walkero-gr** [comment]
@SadieCat Thank you so much for your reply. Now it is clear that I should not keep on trying to make it work. Thank you.
